visitors/table.go

```diff
@@ -19,6 +19,33 @@ func indent(n int) string {
 	return strings.Repeat(" ", n)
 }
 
+// Visit applies the Table visitor to any Firmware type.
+func (v *Table) Visit(f uefi.Firmware) error {
+	switch f := f.(type) {
+	case *uefi.FlashImage:
+		return v.printRow(f, "Image", "", "", "")
+	case *uefi.FirmwareVolume:
+		return v.printRow(f, "FV", f.FileSystemGUID.String(), "", f.Length)
+	case *uefi.File:
+		// TODO: make name part of the file node
+		return v.printRow(f, "File", f.Header.UUID.String(), f.Header.Type, f.Header.ExtendedSize)
+	case *uefi.Section:
+		return v.printRow(f, "Sec", f.Name, f.Type, fmt.Sprintf("%d", f.Header.ExtendedSize))
+	case *uefi.FlashDescriptor:
+		return v.printRow(f, "IFD", "", "", "")
+	case *uefi.BIOSRegion:
+		return v.printRow(f, "BIOS", "", "", "")
+	case *uefi.MERegion:
+		return v.printRow(f, "ME", "", "", "")
+	case *uefi.GBERegion:
+		return v.printRow(f, "GBE", "", "", "")
+	case *uefi.PDRegion:
+		return v.printRow(f, "PD", "", "", "")
+	default:
+		return v.printRow(f, "Err", "", "", "")
```

Comment:
one thing you could do here is instead, print unknown firmware type, and use reflection to display the type. That'll be more useful than just an Err